include/extractor/scripting_environment_lua.hpp

```diff
@@ -33,6 +45,8 @@ struct LuaScriptingContext final
     bool has_node_function;
     bool has_way_function;
     bool has_segment_function;
+
+    location_handler_type * location_handler;
```

Comment:
I'm not sure we want to store the location cache in the Lua scripting environment. From a conceptual standpoint, all scripting environments need the cache. Also we're currently storing the coordinates manually anyway in the extractor. A good place to put this therefore would be the extractor.